exist-docker/src/main/resources-filtered/Dockerfile-DEBUG

```diff
@@ -70,7 +71,7 @@ ENV JAVA_TOOL_OPTIONS \
   -XX:+UseContainerSupport \
   -XX:MaxRAMPercentage=${JVM_MAX_RAM_PERCENTAGE:-75.0} \
   -XX:+ExitOnOutOfMemoryError \
-  -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005
+  -agentlib:jdwp=transport=dt_socket,server=y,suspend=${JVM_JDWP_SUSPEND:-n},address=5005
```

Comment:
Could it be that there is a unintended minus sign?

Reply:
@reinhapa It is intentional. It means if there is no value then use `n`